README.md

```diff
@@ -1,22 +1,17 @@
 # Authentication & Proxy
 ## Team 1 | Pro.To.Type
-- Mason K.
-- Seven A.
-- Joshua H.
-- Cameron B.
-- Owen S.
+- Mason Kuenneke
+- Seven Akridge
+- Joshua Haase
+- Cameron Brooks
+- Owen Starks
 
 ## Setup
-- Open a git bash prompt in the root of the project
-- Run the following commands:
-  - `./mvnw generate-resources`
-  - `npm i` (`npm install`)
-  - `./npm i @angular/cli` (`npm install @angular/cli`)
-  - `./npm i` (`npm install`)
-  - `./mvnw generate-resources`
+- Open a Git Bash prompt in the root of the project
+- Run the following command: `./mvnw generate-resources`
```

Comment:
Nice catch, tested this, with some changes from our original application the other 4 commands are no longer needed